lib/interface_quda.cpp

```diff
@@ -5548,10 +5547,8 @@ void performAPEnStep(unsigned int nSteps, double alpha)
   GaugeFieldParam gParam(*gaugeSmeared);
   auto *cudaGaugeTemp = new cudaGaugeField(gParam);
 
-  if (getVerbosity() == QUDA_VERBOSE) {
-    double3 plq = plaquette(*gaugeSmeared, QUDA_CUDA_FIELD_LOCATION);
-    printfQuda("Plaquette after 0 APE steps: %le %le %le\n", plq.x, plq.y, plq.z);
-  }
+  double3 plaq = plaquette(*gaugeSmeared);
+  printfQuda("Plaquette after 0 APE steps: %le %le %le\n", plaq.x, plaq.y, plaq.z);
```

Comment:
These `printQuda` statements need to have verbosity guards around them since if the verbosity is `QUDA_SILENT` then QUDA should not print anything.  Though the prior ones do look buggy since they didn't use `>=`, perhaps `if (getVerbosity() >= QUDA_SUMMARIZE) { ... }` ?

Reply:
Changed to >= QUDA_SUMMARIZE.